snuba/datasets/storages/groupassignees.py

```diff
@@ -41,7 +41,7 @@
     storage_key=StorageKey.GROUPASSIGNEES,
     storage_set_key=StorageSetKey.EVENTS,
     schema=schema,
-    query_processors=[PrewhereProcessor()],
+    query_processors=[PrewhereProcessor(["project_id"])],
```

Comment:
This is a functional change, group assignees had no prewhere candidates defined earlier